src/network/MissionControlProtocol.cpp

```diff
@@ -176,6 +177,33 @@ void MissionControlProtocol::handleJointPowerRequest(const json& j) {
 	}
 }
 
+void MissionControlProtocol::sendRoverPos() {
+	auto heading = robot::readIMUHeading();
+	auto gps = robot::readGPS();
+	if (gps.isValid() && heading.isValid()) {
+		double orientX = 0.0;
+		double orientY = 0.0;
+		double orientZ = sin(heading.getData() / 2);
+		double orientW = cos(heading.getData() / 2);
```

Comment:
I'd say change `sin` and `cos` for `std::sin` and `std::cos`, respectively. `std` methods tend to play better with types and such, so we prefer to use them.